### PR TITLE
Introduce motivation for link types, explain restrictions for peering links

### PR DESCRIPTION
**Pull request by matzf**
Contributes to #17 (by virtue of avoiding the phrase "peering AS").

### CODE REVIEW

**Comment by matzf** [comment]
As discussed, this is overlapping with #29. 
#29 should be merged first,  then I'll take another look on how to unify the two changes.

**Review by jiceatscion — APPROVED** [review]

**Review by nicorusti — APPROVED** [review]
Thanks a lot, crisp and clear!

File: draft-dekater-scion-controlplane.md [hunk 1]
```diff
@@ -224,11 +222,17 @@ As SCION is an *inter-domain* network architecture, it only deals with *inter*-d
...
 
-In SCION, autonomous systems (ASes) are organized into logical groups called isolation domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (i.e., a common jurisdiction). An ISD is administered by a set of distinguished ASes called core ASes. Within and between ISDs, SCION supports three types of links: (1) core links, (2) parent-child links, and (3) peering links.
+In SCION, autonomous systems (ASes) are organized into logical groups called isolation domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (i.e., a common jurisdiction). An ISD is administered by a set of distinguished ASes called core ASes.
+
+SCION distinguishes three types of links between ASes: (1) core links, (2) parent-child links, and (3) peering links.
+
+- *Core* links connect two core ASes, which are either within the same or in a different ISD. Core links can exist for various underlying business relationships, including provider-customer (where the customer pays the provider for traffic) and peering relationships.
```
Comment (jiceatscion) [inline draft-dekater-scion-controlplane.md:229]: "or in a different ISD" -> "or in different ISDs"

File: draft-dekater-scion-controlplane.md [hunk 2]
```diff
@@ -224,11 +222,17 @@ As SCION is an *inter-domain* network architecture, it only deals with *inter*-d
...
+- *Peering* links exist between ASes with a (settlement-free or paid) peering relationship. They can be established between any two ASes (core or non-core). Peering links can cross ISD boundaries.
 
-- A *core* link always connects two core ASes, which are either within the same or in a different ISD. Core links can exist for various underlying business relationships, including provider-customer (where the customer pays the provider for traffic) and peering relationships.
-- *Parent-child* links create a hierarchy between the parent and the child. ASes with a parent-child link typically have a provider-customer relationship.
-- *Peering* links exist between ASes with a (settlement-free or paid) peering relationship. Peering links can only be used to reach destinations within or downstream of the peering AS. They can be established between any two core or non-core ASes, and between core and non-core ASes. Peering links can also cross ISD boundaries.
+These link types form the basis for the notion of "valley-free" paths. Valley-freedom simply means that a child AS does not carry transit traffic from a parent AS.
```
Comment (jiceatscion) [inline draft-dekater-scion-controlplane.md:233]: "Valley-freedom" sounds weird... it's a tough one, but I found thin precedent for the form "valley-freeness": 

It is interesting to note that there were increment in pulp freeness and reduction in the pulp strength when the same chitosan prepared was added from the first day up to the fifth day.
[Mechanical properties of paper from oil palm pulp treated with chitosan from horseshoe crab](https://www.thefreelibrary.com/Mechanical+properties+of+paper+from+oil+palm+pulp+treated+with...-a0440400148)

**Review by jiceatscion — APPROVED** [review]
...changes at your discretion.

File: draft-dekater-scion-controlplane.md (hunk 2 above)
Comment (matzf) [reply draft-dekater-scion-controlplane.md]: Changed it to valley freeness -- I found one paper using that term, and its from Adrian :smile:.
Also, no hyphen seems correcter.

File: draft-dekater-scion-controlplane.md (hunk 1 above)
Comment (matzf) [reply draft-dekater-scion-controlplane.md]: Done.

**Pull request merged**